internal/proxy/multi_rate_limiter.go

```diff
@@ -314,7 +314,7 @@ func (rl *rateLimiter) registerLimiters(globalLevel bool) {
 		limit := ratelimitutil.Limit(r)
 		burst := r // use rate as burst, because Limiter is with punishment mechanism, burst is insignificant.
 		rl.limiters.GetOrInsert(internalpb.RateType(rt), ratelimitutil.NewLimiter(limit, burst))
-		log.Info("RateLimiter register for rateType",
+		log.RatedInfo(30, "RateLimiter register for rateType",
```

Comment:
Still too much?
We might want to log only if the quota is set and changed in this round

Reply:
updated, info&warn log will be printed if the quota is set and changed, and rated debug log will be printed when quota is normal.